receiver/rabbitmqreceiver/scraper.go

```diff
@@ -0,0 +1,157 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rabbitmqreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/rabbitmqreceiver"
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/model/pdata"
+	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/rabbitmqreceiver/internal/metadata"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/rabbitmqreceiver/internal/models"
+)
+
+const instrumentationLibraryName = "otelcol/rabbitmq"
```

Comment:
Is this somewhere defined that we use instrumentation library name for scrapers? I like this approach, but I believe it's not applied to all scrapers consistently.

Reply:
I found this in the Elasticsearch component I was using as an example. I wasn't aware it's not a convention. Sounds like you're ok keeping it but calling out that there should be some standard put in place?

Reply:
I'm not sure it's defined anywhere but I found this pattern in a couple scrapers that predated my involvement in the collector and have tried to apply it consistently since.

Reply:
Ok. I think this needs to be explicitly defined. Let's discuss it in a separate issue. I will create one.